crates/db_schema/src/impls/local_user.rs

```diff
@@ -115,9 +118,12 @@ impl Crud for LocalUser {
     diesel::delete(local_user.find(local_user_id)).execute(conn)
   }
   fn create(conn: &PgConnection, form: &LocalUserForm) -> Result<Self, Error> {
-    insert_into(local_user)
+    let local_user_ = insert_into(local_user)
       .values(form)
-      .get_result::<Self>(conn)
+      .get_result::<Self>(conn)?;
+    // initialize with all languages
+    LocalUserLanguage::update_user_languages(conn, None, local_user_.id)?;
```

Comment:
Search for references of this, there's another one that should be removed in `api_crud/src/user/create.rs`

Reply:
Good catch, fixed.